src/status_im/utils/ethereum/core.cljs

```diff
@@ -108,6 +110,12 @@
   (when (= 66 (count s))
     (normalized-address (subs s 26))))
 
+(defn coordinates [public-key]
```

Comment:
it is better to always put the param list of the second line, a good way to convince yourself is to write a docstring for this function